docs/getting_started/configuration2.md

```diff
@@ -0,0 +1,94 @@
+```
+
+Then run `helm upgrade` with updated `values.yaml` configuration
+```shell
+helm upgrade pelorus charts/pelorus --namespace pelorus --values values.yaml
+```
+
+To ensure PVC were properly created and Bound to the PV, run
+```shell
+oc get pvc --namespace pelorus
+```
+
+## Configure Long Term Storage (Recommended)
+
+The Pelorus chart supports deploying a thanos (what is this?) instance for long term storage. It can use any S3 bucket provider. The following is an example of configuring a `values.yaml` file for [NooBaa](https://www.noobaa.io/) with the local s3 service name.
+```yaml
+bucket_access_point: s3.pelorus.svc:443
+bucket_access_key: <your access key>
+bucket_secret_access_key: <your secret access key>
+```
+
+The default bucket name is thanos. It can be overridden by specifying the following field.
+```yaml
+thanos_bucket_name: <bucket name here>
+```
+
+Then run `helm upgrade` with updated `values.yaml` configuration:
+```
+helm upgrade pelorus charts/pelorus --namespace pelorus --values values.yaml
+```
+
+If you don't have an object storage provider, we recommend NooBaa as a free, open source option. You can follow our [NooBaa quickstart](../Noobaa.md) to host an instance on OpenShift and configure Pelorus to use it.
+
+## Deploying Across Multiple Clusters
+
+By default, Pelorus will pull in data from the cluster in which it is running, but it also supports collecting data across multiple OpenShift clusters. In order to do this, the thanos sidecar can be configured to read from a shared S3 bucket across clusters. See [Pelorus Multi-Cluster Architecture](../Architecture.md) for details. You define exporters for the desired metrics in each of the clusters and the main cluster's Grafana dashboard will display a combined view of the metrics collected in the shared S3 bucket via thanos.
+
+## Configure Production Cluster
+
+A production configuration example of `values.yaml` with one deploytime exporter, which uses AWS S3 bucket and AWS volume for Prometheus and tracks deployments to production.
+```yaml
+thanos_bucket_name: <bucket name here>
+bucket_access_point: s3.us-east-2.amazonaws.com
+bucket_access_key: <your access key>
+bucket_secret_access_key: <your secret access key>
+
+prometheus_storage: true
+prometheus_storage_pvc_capacity: 20Gi
+prometheus_storage_pvc_storageclass: "gp2"
+
+deployment:
+  labels:
+    app.kubernetes.io/component: production
+    app.kubernetes.io/name: pelorus
+    app.kubernetes.io/version: v0.33.0
+
+exporters:
+  instances:
+  - app_name: deploytime-exporter
+    exporter_type: deploytime
+    env_from_configmaps:
+    - pelorus-config
+    - deploytime-config
+```
+(deployment does not appear in values.yaml, should we add? Add all options so that it can be copied? `openshift_prometheus_htpasswd_auth`, `openshift_prometheus_basic_auth_pass`, `extra_prometheus_hosts`, `prometheus_retention`, `prometheus_retention_size`)
```

Comment:
**deployment** section does not appear in our values.yaml, should we add it?

Should we add all the options so that we can copy the code block to the yaml file? `openshift_prometheus_htpasswd_auth`, `openshift_prometheus_basic_auth_pass`, `extra_prometheus_hosts`, `prometheus_retention` and `prometheus_retention_size`